main/integration-tests/__test__/api-tests/workspace-types/get-workspace-type.test.js

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
I suggest that we add tests for anonymous sessions. There are a few places were we did that, such as the user and study tests.